src/utils/features/copyArticle.ts

```
@@ -0,0 +1,261 @@
+          const linkText = extractFormattedText(el);
+          if (href && href !== '#' && !href.startsWith('#')) {
+            parts.push(`[${linkText}](${href})`);
+          } else {
+            parts.push(linkText);
+          }
+          break;
+        case 'hr':
+          parts.push('\n---\n');
+          break;
+        case 'blockquote':
+          const quoteText = extractFormattedText(el);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)